MultiVersions/Forge/main/beachparty/lang/zh_tw.json

```
@@ -0,0 +1,143 @@
+  "block.beachparty.dry_bush":"乾灌木",
+  "block.beachparty.dry_bush_tall": "高乾灌木",
+  "block.beachparty.empty_sand_bucket": "空沙桶",
+  "block.beachparty.empty_sand_bucket_block": "空沙桶",
+  "block.beachparty.hammock": "沙灘吊床",
+  "block.beachparty.honey_cocktail": "蜂蜜雞尾酒",
+  "block.beachparty.lounge_chair":"棕櫚木躺椅",
+  "block.beachparty.melon_cocktail": "西瓜雞尾酒",
+  "block.beachparty.message_in_a_bottle": "瓶中信",
+  "block.beachparty.mini_fridge": "迷你冰箱",
+  "block.beachparty.notimplemented" : "此方塊的功能尚未完全實現。敬請期待！",
+  "block.beachparty.palm_beam": "棕櫚木樑",
+  "block.beachparty.palm_button": "棕櫚木按鈕",
+  "block.beachparty.palm_door": "棕櫚木門",
+  "block.beachparty.hanging_sign": "懸掛式棕櫚木告示牌",
+  "block.beachparty.palm_fence": "棕櫚木柵欄",
+  "block.beachparty.palm_fence_gate": "棕櫚木柵欄門",
+  "block.beachparty.palm_floorboard": "棕櫚木地板",
+  "block.beachparty.palm_leaves": "棕櫚木樹葉",
+  "block.beachparty.palm_log": "棕櫚木原木",
+  "block.beachparty.palm_planks": "棕櫚木材",
+  "block.beachparty.palm_pressure_plate": "棕櫚木壓力板",
+  "block.beachparty.palm_sapling": "棕櫚木樹苗",
+  "block.beachparty.palm_sign": "棕櫚木告示牌",
+  "block.beachparty.palm_slab": "棕櫚木半磚",
+  "block.beachparty.palm_stairs": "棕櫚木階梯",
+  "block.beachparty.palm_tall_torch": "棕櫚木高火把",
+  "block.beachparty.palm_torch": "棕櫚木火把",
+  "block.beachparty.palm_torch_item": "棕櫚木火把",
+  "block.beachparty.palm_trapdoor": "棕櫚木地板門",
+  "block.beachparty.palm_wall_torch": "牆上的棕櫚木火把",
+  "block.beachparty.palm_wood": "棕櫚木塊",
+  "block.beachparty.pumpkin_cocktail": "南瓜雞尾酒",
+  "block.beachparty.radio": "收音機",
+  "block.beachparty.sand_bucket": "沙桶",
+  "block.beachparty.sand_bucket_block": "沙桶",
+  "block.beachparty.sand_dirty": "帶雜物的沙",
+  "block.beachparty.sand_pile": "沙堆",
+  "block.beachparty.sand_seastars": "帶海星的沙",
+  "block.beachparty.sand_slab": "沙半磚",
+  "block.beachparty.sandcastle": "沙堆城堡",
+  "block.beachparty.sandwaves": "海浪沙",
+  "block.beachparty.stripped_palm_log":"剝皮棕櫚木原木",
+  "block.beachparty.stripped_palm_wood":"剝皮棕櫚木塊",
+  "block.beachparty.sweetberries_cocktail": "草莓雞尾酒",
+  "block.beachparty.table": "棕櫚木桌",
+  "block.beachparty.tiki_bar":"熱帶風格吧檯",
+  "block.beachparty.tiki_chair":"熱帶風格木椅",
+  "entity.beachparty.pelican": "鵜鶘",
+  "entity.minecraft.villager.beachparty.barkeeper": "酒保",
+  "entity.minecraft.villager.beachparty.beach_boy": "沙灘服務生",
+  "gui.beachparty.recipebook.toggleRecipes.refridgable": "可冷藏",
+  "gui.beachparty.recipebook.toggleRecipes.shakeable": "可調製",
+  "item.beachparty.beach_hat": "遮陽帽",
+  "item.beachparty.bikini": "比基尼",
+  "item.beachparty.coconut_open": "剖開的椰子",
+  "item.beachparty.cooked_mussel_meat": "烤扇貝肉",
+  "item.beachparty.cooked_pelican": "烤鵜鶘肉",
+  "item.beachparty.crocs": "鱷魚鞋",
+  "item.beachparty.icecream_cactus": "仙人掌冰棒",
+  "item.beachparty.icecream_chocolate": "巧克力冰淇淋",
+  "item.beachparty.icecream_coconut": "椰子冰棒",
+  "item.beachparty.icecream_melon": "西瓜冰棒",
+  "item.beachparty.icecream_sweetberries": "草莓冰淇淋",
+  "item.beachparty.overgrown_disc": "Overgrown 唱片",
```

一般都是歌曲名不翻譯而已，物品名稱盡量要有翻譯。
這翻譯是隨便想的，可以另外再想一個更合適的。
```suggestion
  "item.beachparty.overgrown_disc": "覆滿草枝的唱片",
```

---

沒細看到，我還以為那是歌曲名 甘溫